src/apps_plugin.c

```diff
@@ -358,28 +358,28 @@ struct pid_stat {
     // uint64_t flags;
 
     // these are raw values collected
-    unsigned long long minflt_raw;
-    unsigned long long cminflt_raw;
-    unsigned long long majflt_raw;
-    unsigned long long cmajflt_raw;
-    unsigned long long utime_raw;
-    unsigned long long stime_raw;
-    unsigned long long gtime_raw; // guest_time
-    unsigned long long cutime_raw;
-    unsigned long long cstime_raw;
-    unsigned long long cgtime_raw; // cguest_time
+    usec_t minflt_raw;
+    usec_t cminflt_raw;
+    usec_t majflt_raw;
+    usec_t cmajflt_raw;
+    usec_t utime_raw;
+    usec_t stime_raw;
+    usec_t gtime_raw; // guest_time
+    usec_t cutime_raw;
+    usec_t cstime_raw;
+    usec_t cgtime_raw; // cguest_time
 
     // these are rates
-    unsigned long long minflt;
-    unsigned long long cminflt;
-    unsigned long long majflt;
-    unsigned long long cmajflt;
-    unsigned long long utime;
-    unsigned long long stime;
-    unsigned long long gtime;
-    unsigned long long cutime;
-    unsigned long long cstime;
-    unsigned long long cgtime;
+    usec_t minflt;
+    usec_t cminflt;
+    usec_t majflt;
+    usec_t cmajflt;
+    usec_t utime;
+    usec_t stime;
+    usec_t gtime;
+    usec_t cutime;
+    usec_t cstime;
+    usec_t cgtime;
```

Comment:
These variables measure utilization that is not `usec_t`. They are whatever the system provides (`flt` are page faults, `time` are CPU time in Jiffies or whatever the system uses).